readthedocs/notifications/messages.py

```diff
@@ -142,7 +140,7 @@ def get_rendered_body(self):
     ),
     Message(
         id=BuildUserError.SKIPPED_EXIT_CODE_183,
-        header=_("Build skipped manually."),
+        header=_("Build skipped programmatically."),
```

Comment:
While `programmatically` is definitely more technically correct, as a user I wouldn't be sure if `programmatically` refers to something in the RTD application or something in the project's configuration.

I think `manually` is actually more clear, but I see how `manually` also feels like it refers to a user clicking a button to "manually" skip.

This would benefit from more specific language:

```suggestion
        header=_("Build skipped with command exit code 183."),
```

Reply:
I wanted to differentiate "Cancelled manually by clicking the `Cancel` button" from "Cancelled programmatically because a command exited with 183"

Also, I thought that exposing "with command exit code 183" in the header is too verbose and it was enough by communicating that only in the body.

Reply:
Good point, I didn't see the body text on this change. I think specificity still works best here, compared to `programmatically` which is quite vague. Agreed we don't need to be verbose here, perhaps something closer to:

```suggestion
        header=_("Build skipped by exit code."),
```

Maybe even just `Build skipped` and avoid this altogether. The body has the details on why the build was skipped.